### ISSUE TITLE
[Feature] Cropping improvements

### ISSUE THREAD

**Issue by Romern** [Ethereal-Developers-Inc/OpenScan#25]
In OfficeLens, the corner-nodes in cropping can be adjusted by clicking in the general direction, I think the closest point will then be dragged. This is useful, as if you have to directly drag the nodes, you cannot see exactly where you place them while you adjust them. Also, a zoom option would be nice to crop exactly in the corners, even on smaller screens.

**Comment by test2a** [comment]
#14 this was already rejected as wontfix. Sad.

**Comment by Romern** [comment]
@test2a the issue you linked seems unrelated? This is about the process of cropping the image to a rectangle, yours seems to be about the output of the pdf.

**Comment by test2a** [comment]
Oh. Yeah.

**Comment by natrius** [comment]
I feel like both https://github.com/Ethereal-Developers-Inc/OpenScan/issues/48 and this one would be good. Maybe make on default and the other one optional, if possible and the developing time is acceptable for both solutions. 

If its this one it could be indicated somehow like an outer ring with a higher opacity.

**Comment by yephny** [comment]
Agreed, this is the most important feature that open source document scanners on Android all lack. Anyone who could implement this, which does not seem to be more challenging than recognizigng shapes and auto-cropping (what a lot of these apps are readily doing).
